include/seqan/sequence/string_set_base.h

```diff
@@ -1598,10 +1598,10 @@ concat(StringSet<TString, TSpec> const & constMe)
 }
 
 template <typename TStrings, typename TDelim>
-inline String<typename Value<typename Value<TStrings>::Type>::Type>
+inline typename Value<TStrings>::Type
```

Comment:
When is this specialization called? For a String of Strings? And what does Concatenator<TSTrings> return?